backend/dev/scripts/wait-for-it/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
No need to review this file, was copied from the web :)

Reply:
Got it !